### PR TITLE
Added fixed go-chi example

### PR DESCRIPTION
**Pull request by tobias-kuendig**
This fixes the example removed in #31 

For static files to be accessible, we need to register a catch-all route in go-chi: `/debug/statsviz/*`

The statsviz GUI is now accessible via `http://localhost:8080/debug/statsviz/`

### CODE REVIEW

File: _example/chi/main.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"time"
+
+	"github.com/arl/statsviz"
+	"github.com/go-chi/chi"
+)
+
+func main() {
+
+	// Force the GC to work to make the plots "move".
+	go work()
```
Comment (arl) [inline _example/chi/main.go:17]: To keep all examples as small and uniform as possible, there's a `work` function in the `_example` package so you don't need to copy it here.

Can you check other examples out and import `work` please?
https://github.com/arl/statsviz/blob/5ea9f0cf1bd172b759c0b7b8bd2f790275ac3ec4/_example/root/main.go#L13

`example` is imported as:
https://github.com/arl/statsviz/blob/5ea9f0cf1bd172b759c0b7b8bd2f790275ac3ec4/_example/root/main.go#L3-L9

**Review by arl — CHANGES_REQUESTED** [review]
Thanks a lot Tobias.

2 things:
 - can we use the `work` function that is in the `example` package and remove the function from `main.go`?
 - I know that http://localhost:8080/debug/statsviz/ works. But do you know if there's a way for http://localhost:8080/debug/statsviz (no trailing slash) to also work? It's not super important but it would be nice to have I guess. However if that's not possible or if that would make the code too much complex let's forget about it. In that case could you we a `fmt.Println` before `http.ListenAndServe` saying `point your browser at http://localhost:8080/debug/statsviz/` or something?

File: _example/chi/main.go (hunk 1 above)
Comment (tobias-kuendig) [reply _example/chi/main.go]: Makes sense! I did not see this in the other examples.

**Comment by tobias-kuendig** [comment]
> But do you know if there's a way for http://localhost:8080/debug/statsviz (no trailing slash) to also work?

I did spend some time on this but did not find an easy solution. Somehow the trailing slash and the `http.FileServer` don't play nice with each other.

What do you think about this solution (added in the latest commit):

```go
    r.Get("/debug/statsviz", func(w http.ResponseWriter, r *http.Request) {
		http.Redirect(w, r, "/debug/statsviz/", 301)
	})
```

**Comment by arl** [comment]
> What do you think about this solution (added in the latest commit)
Ok thank you.

**Review by arl — APPROVED** [review]

**Pull request merged**